### PR TITLE
Add some documentation about MapFishPrint v3

### LINKED ISSUE #1345: MapFish print 3
Is there somewhere a "howto" for migrating from MapFish print v2 to v3 ?

The only thing I found was <a href="https://github.com/camptocamp/c2cgeoportal/tree/master/c2cgeoportal/scaffolds/create/print" target=_blank>here</a>. I also saw that CGXP should be ready for MapFish print v3...

I could try to do something, because the proposed template is quite "light", but as anybody already done something more elaborated ?

### PR DESCRIPTION
**Pull request by sbrunner**
Will fix #1345 ?

### CODE REVIEW

File: doc/integrator/print.rst [hunk 1]
```diff
@@ -19,8 +19,38 @@ list of options the plugin can receive.
 
 *The main viewer includes a Print plugin by default.*
 
-MapFish Print configuration
----------------------------
+Start with MapFish Print v3
```
Comment (asaunier) [inline doc/integrator/print.rst]: Is this title relevent? Why "start"? What about something like "Using MapFish Print v3"?

File: doc/integrator/print.rst [hunk 2]
```diff
@@ -19,8 +19,38 @@ list of options the plugin can receive.
...
+----------------------------
+
+Start the migration by instanciating the default template:
+
+.. prompt::
+
+   cp -r print ~
```
Comment (asaunier) [inline doc/integrator/print.rst]: Where does the "print" dir come from?

Comment (sbrunner) [inline doc/integrator/print.rst]: From the project ...

**Comment by asaunier** [comment]
> From the project ...

Yes but adding something like 
```
cd <project root dir>
```
would be more explict.

Else looks good.

**Pull request merged**